src/sentry/lang/javascript/processor.py

```diff
@@ -187,24 +187,44 @@ def fetch_release_file(filename, release):
     cache_key = 'releasefile:v1:%s:%s' % (
         release.id,
         md5(filename).hexdigest(),
-    )
+    ),
+
+    filename_path = None
+    try:
+        # Can we guarantee filename is always parsable?
+        parsed_url = urlparse(filename)
+        filename_path = parsed_url.path
```

Comment:
We should add the query string back to this. We want the same behavior WRT query strings for now. (Separate conversation, see discussion we had about this on Slack)